www/grid_view/src/module/changeinfo.tpl.jade

```diff
@@ -0,0 +1,38 @@
+div(style="width:100%;")
+    div.no-select(style="width:100%;", ng-click="change.show_details = !change.show_details")
+        a(ng-if="change.revlink", ng-href="{{change.revlink}}", uib-tooltip="{{ change.when_timestamp | timeago}}")
+            | {{ change.comments.split("\n")[0] }}
+        span(ng-if="!change.revlink", uib-tooltip="{{ change.when_timestamp | timeago}}")
+            | {{ change.comments.split("\n")[0] }}
+        | &nbsp;
+        i.fa.fa-chevron-circle-right.rotate(ng-class="{'fa-rotate-90':change.show_details}")
+    div.anim-changedetails(ng-show="change.show_details")
```

Comment:
this looks very much like the changedetails https://github.com/buildbot/buildbot/blob/master/www/base/src/app/common/directives/changedetails/changedetails.tpl.jade#L23

We need to have a single directive to display a change